packages/core/bootstrap/src/lib/middleware/burst-limit/index.ts

```diff
@@ -22,14 +22,21 @@ const availableSecondLimitCapacity = async (
   store: Store<BurstLimitState>,
   burstCapacity1s: number,
 ) => {
+  const enabled = parseBool(getEnv('RATE_LIMIT_CAPACITY'))
+  const defaultSecondslimit = 429
+
   for (let retry = SECOND_LIMIT_RETRIES; retry > 0; retry--) {
     store.dispatch(actions.updateIntervals())
     const state = store.getState()
     const { requests }: { requests: RequestsState } = state
 
     const observedRequestsInSecond = selectTotalNumberOfRequestsFor(requests, IntervalNames.SECOND)
 
-    if (observedRequestsInSecond > burstCapacity1s) {
+    if (enabled && observedRequestsInSecond >= defaultSecondslimit) {
+      logger.debug(
+        `Per Second Burst rate limit cap of ${defaultSecondslimit} reached. ${observedRequestsInSecond} requests sent in the last minute. Waiting 1 second. Retry number: ${retry}`,
```

Comment:
This message should be more different from the original log message. We need to clearly call out that the user has configured a custom rate limit capacity.